### DISCUSSION TITLE
Inclined brakes — which ride types need them?

### DISCUSSION THREAD

**Discussion by fidwell** [OpenRCT2/OpenRCT2#22405] [Feature requests]
If inclined brakes (#22383) become a thing, it would be helpful to figure out which ride types in the game would benefit from them the most. As of now, we only have sprites for the hybrid and single-rail gentle slopes.

| Track type | 25° up | 25° down | 60° up | 60° down | 90° down |
| ---------- | ----- | ------- | ----- | ------- | ------- |
| Box-spine (B&M) | [Shambahala](https://youtu.be/yiHuAok3YL4?t=73) | common on B&M brake runs. [Diamondback](https://youtu.be/XJR9ZaW1MYg?t=120) [Orion](https://youtu.be/UP1M5vOkkxs?t=100) | extremely common on B&M hypers. [Diamondback](https://youtu.be/XJR9ZaW1MYg?t=79) [Apollo's Chariot](https://youtu.be/a0inH_zqst0?t=89) | [Rougarou](https://youtu.be/RuweAtyeUuU?t=68) | |
| Double-spine (Intamin) | | [I305](https://youtu.be/eBHyhKO6VPY?t=98) | [I305](https://youtu.be/eBHyhKO6VPY?t=81)* | | |
| Wooden | [Thunderhawk](https://youtu.be/Fcxkn1EttDA?t=87) | | | [Beast](https://youtu.be/aSAt9eAqLkI?t=70) | |
| LIM / looping | | [Full Throttle](https://youtu.be/4Kb7K9KsB7o?t=61), [Abyssus](https://rcdb.com/17412.htm#p=126632) | | [Full Throttle](https://youtu.be/4Kb7K9KsB7o?t=61) | [Full Throttle](https://youtu.be/4Kb7K9KsB7o?t=61) |
| Hybrid | | [Iron Rattler](https://youtu.be/Wy5LraGCrWE?t=122) | | | |
| Arrow-style | [Steel Phantom](https://youtu.be/_kbNv3vBvcI?t=29) | | | | |
| Single-rail | | | | | |

*debatably 25 up

(I think we could reuse the hybrid's brake sprites on some of the other coaster types (box spine and wooden) with minor modifications and they'd look pretty good.)

Anyway, if you know of any real-life coasters that would fill in this chart, please let me know, so we have a better idea of what ride types would most benefit from these track pieces!

**Comment by astraylife1** [comment]
Magnum's trims are in a valley. I would consider them 0º or even downward. Not sure if you meant to place an asterisk next to Magnum. Mantis had trim brakes at 60º down on the first drop, though I can't see anyone realistically using this in RCT.
![642-2614](https://github.com/user-attachments/assets/53f2dc4b-3f53-49bd-a27a-0dd269bd973b)

**Reply by fidwell** [reply]
You're right; looking again I'd call the Magnum ones flat. The banking threw me a little.

Rougarou still has the trims on the first drop so I'm definitely counting that.

**Comment by karst** [comment]
I'd like to introduce you to full throttle, the biggest buzzkill brake of all time. (LIM Launched rollercoaster in the game)
This has a brake on 90º down, 60º down, 25º down and flat.
![sfmm_full_throttle_1440x1533](https://github.com/user-attachments/assets/4b01ce3f-7f38-46e6-aa0b-83f4175f0f75)

**Comment by swNero** [comment]
[Abyssus](https://rcdb.com/17412.htm) has some trim brakes on its top as well, but this would be rather on a "Flat to 25 Down" piece.

**Reply by fidwell** [reply]
I could see an argument either way. Interesting find!

**Comment by brisingr25** [comment]
There's also the top hat ''anticipation'' brakes like those on Toutatis or Batman Gotham City Escape, which would have to be on a sloped to steep piece and steep to vertical piece, I think. Similar to the vertical drop coaster's holding brakes but without stopping the train.

![cats](https://github.com/user-attachments/assets/2533ab16-1411-45af-b8cd-4bca1273333d)

**Comment by brisingr25** [comment]
B&M gentle up brakes should also be included, like the trim brakes in Shambhala's speed hill. Although this would preferably be in a diagonal gentle up as straight 25º speed hills look too triangular.

![trims](https://github.com/user-attachments/assets/9b6af889-622d-4d61-936e-06765facc94d)

**Reply by fidwell** [reply]
Nice find. If the diagonal brakes get approved (hopefully soon), my goal is to add them for all coaster types we can make sprites for, so diagonal _and_ orthogonal.

**Comment by SpartanFrederic104** [comment]
Don't know if this counts, but [Steel Phantom](https://www.youtube.com/watch?v=_kbNv3vBvcI) (a former Arrow looping/corkscrew coaster) featured an upward inclined brake right before the vertical loop. (at the 30 sec mark in the video; excuse the low video quality)
